src/components/API.js

```diff
@@ -25,7 +25,7 @@ export default class Api {
       },
```

Comment:
In **Files changed** there should be the code that should be reviewed. So, all changes from the last sprint should be there. But now you have only 1 file there. I'm not able to leave comments in your code. Please, try to send me a new pull request with all files changed

# So, I'll leave comment just here


* See here https://disk.yandex.ru/i/Wpp3oU3zV9X6EQ you need to pass `validationOptions` into the constructor rather than hardcode them inside the class
* See here https://disk.yandex.ru/i/sTo8EsItek4GCA `setLoading` won't work in line 24. You set the text that the button already has
* You shouldn’t create the **same methods** of the parent class if you don’t change it inside and only call it with `super`. The parent methods are available in any inheritor of the class. See here https://disk.yandex.ru/i/ujgJEkCT79Nl3g